tools/generator/test/Fixtures.swift

```diff
@@ -1562,8 +1562,8 @@ cp "${SCRIPT_INPUT_FILE_0}" "${SCRIPT_OUTPUT_FILE_0}"
             "E1": [
                 PBXSourcesBuildPhase(
                     files: buildFiles([PBXBuildFile(
-                        file: elements[.external("a_repo/a.swift")]!),
-                    ])
+                        file: elements[.external("a_repo/a.swift")]!
+                    )])
```

Comment:
@brentleyjones I removed the weird comma for now. This passes `swiftformat`. We will deal with the `swiftlint` issues in subsequent PRs.